src/instances/RaidInstance.ts

```diff
@@ -501,6 +529,9 @@ export class RaidInstance {
      */
     public static async createNewLivingInstance(guildDoc: IGuildInfo,
                                                 raidInfo: IRaidInfo): Promise<RaidInstance | null> {
+        const logger = new Logger(__filename, false);
```

Comment:
Maybe one static `Logger` for every `RaidInstance` is good enough (instead of having two `Logger` objects be created). Same idea w/ `HeadcountInstance`